packages/firmware-release-config/src/types.ts

```diff
@@ -32,10 +32,9 @@ export interface ReleasesConfig {
     T3W1: ConditionalRelease[];
 }
 export interface IntermediaryReleaseConfig {
-    if_version_less_than: string;
+    min_firmware_version: VersionArray;
```

Comment:
btw, can't the above type (ReleseCOnfig) be defined this way?

```
export type ReleasesConfig = Record<DeviceModelInternal, ConditionalRelease[]>;
```
so that when a future model comes, you don't need to add it to the definition manually?

Reply:
Sure, I changed it.